docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3239,72 +3239,72 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0153E":"(%s) is not a supported platform for %s. Skipping (%s)... Supported: %s
+### ZWED0153E":"(_operatingSystemName_) is not a supported platform for _componentName_. Skipping (_pluginid_)... Supported: _requiredOperatingSystem_
 
   **Reason:**
 
-  TODO
+  The Zowe host operating system is not a supported by the component. Supported platforms are defined in the component's pluginDefinition.json.
 
   **Action:**
 
-  TODO
+  Refer to _componentName_ pluginDefinition.json for supported platforms. The installation of Zowe may also be moved to a supported platform. Lastly, contact the author of the component, or a system administrator.
 
 
 
-### ZWED0154E":"(%s) is not a supported architecture for %s. Skipping (%s)... Supported: %s
+### ZWED0154E":"(_architectureName_) is not a supported architecture for _componentName_. Skipping (_pluginid_)... Supported: _requiredArchitecture_
 
   **Reason:**
 
-  TODO
+  The Zowe host architecture is not supported by _componentName_. Supported architectures are defined in the component's pluginDefintion.json.
 
   **Action:**
 
-  TODO
+  Refer to _componentName_ pluginDefinition.json for supported architectures. The installation of Zowe may also be moved to a supported architecture. Lastly, contact the author of the component, or a system administrator.
 
 
 
-### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
+### ZWED0155E":"(_url_) is not a supported endpoint for _componentName_. Skipping (_pluginid_)... Supported: _urls_
 
   **Reason:**
 
-  TODO
+  The endpoint _url_ does not match any required endpoints of _componentName_. Supported endpoints may be viewd in the component's pluginDefinition.json.
 
   **Action:**
 
-  TODO
+ Refer to _componentName_ pluginDefinition.json for supported endpoints. Optionally, remove _url_ from the required endpoints in pluginDefinition.json. Lastly, contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0156E":"Could not register default plugins into app-server
```

Comment:
I'm having a hard time finding where we use this error code, do we use it?

Reply:
Yes, try refining your search within the core zowe components to search for the numeric part of the error code + the severity character and exclude your search to js, ts, jsx, and tsx files.